api/apierr/service.go

```diff
@@ -0,0 +1,27 @@
+package apierr
```

Comment:
Has this been reverted from `error` on purpose?

Reply:
“error” is a reserved word for the ERROR type

Reply:
Ahh I see.